src/Utilities/InternetParser.cs

```diff
@@ -1,32 +1,29 @@
 ﻿using System;
-using System.IO;
+using System.Diagnostics;
 using System.Linq;
 using System.Net;
 using System.Net.Http;
 using System.Threading.Tasks;
+using ePiggyWeb.DataManagement.Goals;
 using HtmlAgilityPack;
-
 namespace ePiggyWeb.Utilities
 {
     public static class InternetParser
     {
-        /*Example of calling method:
-         * Task.Run(() => InternetParser.GetHTMLAsync()).Wait();
-        */
-        private static readonly string ResourceDirectoryParsedGoal = Directory.GetParent(Environment.CurrentDirectory)
-                                                                         .Parent?.Parent?.FullName +
-                                                                     @"\resources\textData\parsedGoal.txt";
-        public static async Task ReadPriceFromCamel(string itemName)
+        private static readonly Func<int, string, bool> IsTooLong = (int x, string s) => s.Length > x;
+        private static readonly Func<decimal, decimal> ConvertToEur = (decimal price) => price * (decimal) 1.11;
```

Comment:
Explicit declaration of types is unnecessary here, could just be (x, s) instead of (int x, string s)